### DISCUSSION TITLE
How to keep several major versions updated?

### DISCUSSION THREAD

**Discussion by ablekhman** [renovatebot/renovate#8878] [Request Help] [answered]
We have a Maven-based Java project that uses several major versions of some dependency. It is intentional because these versions used in integration tests to ensure compatibility with previous releases.

Once Renovate bot sees these dependencies, it tries to update all of them to the latest major.minor version. I would like to keep several **major** releases concurrently, while updating their respective minor+patch versions. Example:

**project-module-1**:
```xml
<dependency>
    <groupId>com.example</groupId>
    <artifactId>some-lib</artifactId>
    <version>1.2.1</version>
</dependency>
```

**project-module-2**:
```xml
<dependency>
    <groupId>com.example</groupId>
    <artifactId>some-lib</artifactId>
    <version>2.4.3</version>
</dependency>
```
I would like to keep both 1.x **and** 2.x lines updated. Is it possible?

**Comment by rarkins** [comment, marked as answer]
By default Renovate will propose both major and non-major updates for each existing dependency. You can close the major update PR for 1.x to ignore it and still get non major updates in future.
